src/Display.qml

```diff
@@ -21,42 +22,77 @@ import org.asteroid.controls 1.0
 import org.asteroid.utils 1.0
 import "calculator.js" as CalcEngine
 
-Label {
+Text {
```

Comment:
Same comment as in another PR, you should pretty much always use Labels instead of Text